sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -603,22 +621,22 @@ case class DescribeTableCommand(
     }
     table.storage.toLinkedHashMap.foreach(s => append(buffer, s._1, s._2, ""))
   }
+}
 
-  private def describeSchema(
-      schema: StructType,
-      buffer: ArrayBuffer[Row],
-      header: Boolean): Unit = {
-    if (header) {
-      append(buffer, s"# ${output.head.name}", output(1).name, output(2).name)
-    }
-    schema.foreach { column =>
-      append(buffer, column.name, column.dataType.simpleString, column.getComment().orNull)
-    }
-  }
+/**
+ * Command can of the following form.
+ * {{{
+ *   DESCRIBE [QUERY] query_statement
```

Comment:
`query_statement` -> `select_statement`?

Also, if possible, let's add some comment below line 627 that we don't support CTE syntax yet.

Reply:
@dongjoon-hyun Now we support statements of the form : 
1) SELECT
2) SELECTS wrapped under a set op (union/intersect etc)
3) VALUES(....)
4) TABLE <tabName>
Thats why i didn't use select_statement.. Any other name you can think of ?

Reply:
I got it. BTW, `TABLE` and `VALUES` doesn't look like query statement. What about enumerating them all clearly?